source/CAS/Autoload.php

```diff
@@ -31,7 +31,7 @@ function CAS_autoload($class)
     }
     // Setup the include path if it's not already set from a previous call
     if (empty($include_path)) {
-        $include_path = array(dirname(__DIR__), dirname(__DIR__) . '/../test/' );
+        $include_path = array(dirname(__DIR__));
```

Comment:
Is this being removed because tests will only run with composer (because of the new dependencies)?

Reply:
Yes, that is right. This moves to a single method to bootstrap and run the test environment: composer. I've been using the following:

```
$ composer install
$ ./vendor/bin/phpunit
```

Reply:
To add, through the PSR-4 configuration, composer automatically enables class loading for the tests. It no longer needs to be handled manually by phpCAS.